Chips/Chips.jsx

```diff
@@ -40,6 +40,18 @@ class Chip extends MaterialComponent {
   }
 }
 
+class ChipIcon extends MaterialComponent {
```

Comment:
this should extend `Icon`.
It brings `material-icons` classes with it

https://github.com/material-components/material-components-web/tree/master/packages/mdc-chips#trailing-icon